test/test_config.py

```diff
@@ -0,0 +1,44 @@
+from tempfile import NamedTemporaryFile
+
+from simulation.config import SimulationConfig
+
+
+def test_config_defaults():
+    config = SimulationConfig()
+    assert config.getboolean('simulation', 'validate') is True
```

Comment:
I read about getboolean online and I don't quite understand how it works and what is going on here.